src/transformers/modeling_tf_bart.py

```diff
@@ -1130,15 +1199,17 @@ def adjust_logits_during_generation(self, logits, cur_len, max_length):
         return logits
 
     @staticmethod
-    def _force_token_id_to_be_generated(scores, token_id) -> None:
+    def _force_token_id_to_be_generated(scores, token_id, inverted=False) -> None:
```

Comment:
why do we need the flag `inverted` here?

Reply:
I see `Blenderbot` probably... this function is really not the easiest to understand and tweaking it to make it use for other models is not the way to go here IMO...